apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -690,6 +718,74 @@ private void transformEntitlementDualEligibility(Coverage coverage, Beneficiary
         coverage, CcwCodebookVariable.DUAL_12, beneficiary.getMedicaidDualEligibilityDecCode());
   }
 
+  /**
+   * current monthly Medicare-Medicaid dual eligibility code extensions to the provided {@link
+   * Coverage} resource.
+   *
+   * @param coverage the {@link Coverage} to generate
+   * @param beneficiary the {@link Beneficiary} to generate Coverage for
+   * @param currentMonth the value of current month in integer
+   */
+  private void currentMonthTransformEntitlementDualEligibility(
```

Comment:
This isn't a big deal, so feel free to ignore, but shouldn't the verb come first in the method name?